docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
+       - An issue with trying to create a new user that shares the same user name with a previous user that was removed from the system failing has been resolved.
+       - A problem with converting very long integers from JSON has been resolved. This would have manifested itself as a crash when handling datapoints that contain 64 bit integers above a certain value.     
+       - An update has been made to prevent the creation of services with empty service names
+       - An issue that could cause health metrics to not be correctly returned when using the Postgres storage engine has been resolved.
 
 
 - **GUI**
 
     - New Features:
 
-       - 
-       - 
-       - New controls have been added to the asset browser to apse the automatic refresh of the data should and to allow shuffling back and forth along the timeline.
+       - New controls have been added to the asset browser to pause the automatic refresh of the data and to allow shuffling back and forth along the timeline.
        - New controls have been added in the menu pane of the GUI to allow nested commands to be collapsed or expanded, resulting in a smaller menu display.
        - The ability to select an area on the graph shown in the asset browser and zoom into the time period defined by that area has been added.
-       - The interface for updating the filters has been improved when multiple filters are being updated at once.
-       - 
-       - The display of image attributes for image type data points has been added to the latest reading display.
        - The reading graph time granularity has been improved in the asset browser.
-       - A new user interface option has been added to the control menu to create control pipelines.
        - The ability to move backwards and forwards in the timeline of the asset browser graph has been added.
-       - The facility pause the automatic update of the asset browser graph has been added.
+       - The facility that pauses the automatic update of the asset browser graph has been added.
        - The ability to graph multiple readings on a single graph has been added to the asset browser graph.
        - A facility to allow a user to define the default time duration shown in the asset browser graph has been added to the user interface settings page.
-       - The user interface has been updated such that if the backend system is not available then the user interface components are made insensitive.
        - The date format has been made more flexible in the asset and readings graph.
+       - A new user interface option has been added to the control menu to create control pipelines.
+       - The user interface has been updated such that if the backend system is not available then the user interface components are made insensitive.
```

Comment:
"...user interface components are made insensitive." What does _insensitive_ mean in this context? Grayed out? Case insensitive?

Reply:
This has not been updated.